spec/models/social_networking/on_the_mind_statement_spec.rb

```diff
@@ -45,5 +46,41 @@ module SocialNetworking
 
       expect(otms.shared_description).to eq("I'm writing tests.")
     end
+
+    describe "when a statement exists" do
```

Comment:
@ericcf test driven development would have caught that.  Okay, all set.  Thanks!